docs/errors.rst

```diff
@@ -125,21 +125,25 @@ Group 0 - Connection
 
 Related to the connection with the database.
 
-+---------------+-------------------------------------------------------------+
-| Code          | Description                                                 |
-+===============+=============================================================+
-| .. _pgrst000: | Could not connect with the database due to an incorrect     |
-|               | :ref:`db-uri` or due to the PostgreSQL service not running. |
-| PGRST000      |                                                             |
-+---------------+-------------------------------------------------------------+
-| .. _pgrst001: | Could not connect with the database due to an internal      |
-|               | error.                                                      |
-| PGRST001      |                                                             |
-+---------------+-------------------------------------------------------------+
-| .. _pgrst002: | Could not connect with the database when building the       |
-|               | :ref:`schema_cache` due to the PostgreSQL service not       |
-| PGRST002      | running.                                                    |
-+---------------+-------------------------------------------------------------+
++---------------+-------------+-------------------------------------------------------------+
+| Code          | HTTP status | Description                                                 |
++===============+=============+=============================================================+
+| .. _pgrst000: | 503         | Could not connect with the database due to an incorrect     |
+|               |             | :ref:`db-uri` or due to the PostgreSQL service not running. |
+| PGRST000      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst001: | 503         | Could not connect with the database due to an internal      |
+|               |             | error.                                                      |
+| PGRST001      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst002: | 503         | Could not connect with the database when building the       |
+|               |             | :ref:`schema_cache` due to the PostgreSQL service not       |
+| PGRST002      |             | running.                                                    |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst003: | 503         | The request time exceeded the timeout specified in          |
+|               |             | :ref:`db-pool-acquisition-timeout`.                         |
+| PGRST003      |             |                                                             |
```

Comment:
Should be merged after this config is added to the docs, I'll update with the PR# when it's ready.

Reply:
Configuration parameter added in PR #591.